### PR TITLE
Fix sync errors in timestamp_queries and hdr samples

### PR DESCRIPTION
**Pull request by Michal-Masiukiewicz-Mobica**
## Description

This PR fixes synchronization errors produced by validation layers in 'timestamp_queries' and 'hdr' samples.
Fixes for both samples are the same as this samples contain similar code.
With this fixes there are no more validation errors.

## Notes:

Tested on:
VulkanSDK: 1.3.239.0
Driver Version: 528.49.0.0
Windows 10
Nvidia Quadro P600
Intel i7-8750H
32 GB RAM

## General Checklist:

Please ensure the following points are checked:

- [x] My code follows the [coding style](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Code-Style)
- [x] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Copyright-Notice-and-License-Template)
- [x] I have updated the header Copyright to reflect the current year (CI build will fail if Copyright is out of date)
- [x] I have commented any added functions (in line with Doxygen)
- [x] I have commented any code that could be hard to understand
- [x] My changes do not add any new compiler warnings
- [x] My changes do not add any new validation layer errors or warnings
- [x] I have used existing framework/helper functions where possible
- [ ] My changes build on **Windows**, Linux, macOS and Android. Otherwise I have [documented any exceptions](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#General-Requirements)
- [x] My changes do not add any regressions
- [ ] I have tested every sample to ensure everything runs correctly
- [x] This PR describes the scope and expected impact of the changes I am making

## Sample Checklist

If your PR contains a new or modified sample, these further checks must be carried out *in addition* to the General Checklist:
- [x] I have tested the sample on at least one compliant Vulkan implementation
- [x] If the sample is vendor-specific, I have [tagged it appropriately](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#General-Requirements)
- [x] I have stated on what implementation the sample has been tested so that others can test on different implementations and platforms
- [x] Any dependent assets have been merged and published in downstream modules
- [x] For new samples, I have added a paragraph with a summary to the appropriate chapter in the [samples readme](./../samples/README.md)

### CODE REVIEW

**Review by gary-sweet — DISMISSED** [review]

**Comment by asuessenbach** [comment]
I can confirm, no more validation errors :)

Would you mind porting these changes to hpp_hdr and hpp_timestamp_queries as well?

**Comment by Michal-Masiukiewicz-Mobica** [comment]
@asuessenbach @SaschaWillems @gary-sweet  I have updated the review. Can you check it now?

File: samples/api/hpp_timestamp_queries/hpp_timestamp_queries.cpp [hunk 1]
```diff
@@ -311,7 +311,7 @@ void HPPTimestampQueries::render(float delta_time)
 		update_uniform_buffers();
 }
 
-void HPPTimestampQueries::create_attachment(vk::Format format, vk::ImageUsageFlagBits usage, FramebufferAttachement *attachment)
+void HPPTimestampQueries::create_attachment(vk::Format format, vk::ImageUsageFlagBits usage, FrameBufferAttachment *attachment)
```
Comment (asuessenbach) [inline samples/api/hpp_timestamp_queries/hpp_timestamp_queries.cpp]: This is not a typo, the data structure should be named `FramebufferAttachment`, not `FrameBufferAttachment`, as it's an Attachment to a Framebuffer.
If you will, it's a typo in the timestamp_queries sample.

**Review by asuessenbach — COMMENTED** [review]
Thanks for adjusting the hpp-based samples!

File: samples/api/hpp_timestamp_queries/hpp_timestamp_queries.cpp (hunk 1 above)
Comment (Michal-Masiukiewicz-Mobica) [reply samples/api/hpp_timestamp_queries/hpp_timestamp_queries.cpp]: Sorry for mistake, reverted this rename.

**Review by asuessenbach — DISMISSED** [review]

**Review by gary-sweet — DISMISSED** [review]

**Review by SaschaWillems — DISMISSED** [review]

**Pull request merged**